client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx

```diff
@@ -36,7 +33,7 @@ const GoogleAppsDetails = () => {
 						),
 					},
 					args: {
-						email: user.get().email,
+						email: user().get().email,
```

Comment:
This component should also use Redux.

(and also the `useTranslate` hook instead of calling `i18n.translate`, but that's unrelated)

Reply:
Done!